.storybook/3.COLORS.stories.mdx

```diff
@@ -0,0 +1,182 @@
+import { Meta } from '@storybook/addon-docs';
+import ActionaleMessage from '../ui/components/ui/actionable-message';
+import designTokenDiagramImage from './images/design.token.graphic.svg';
+
+<Meta title="Design Tokens / Color" />
+
+# Color
+
+Color is used to express style and communicate meaning.
+
+<ActionaleMessage
+  type="warning"
+  message="We are in the process of consolidating all of our colors, making them accessible and enabling theming. Many of the colors used throughout the codebase are deprecated please follow the guide below to ensure you are using the correct colors when building MetaMask UI"
+/>
+
+<br />
+
+## Design Tokens
+
+We are importing design tokens as CSS variables from [@metamask/design-tokens](https://github.com/MetaMask/design-tokens) repo to help consolidate colors and enable theming across all MetaMask products.
+
+### Token Tiers
+
+We follow a 3 tiered system for color design tokens and css variables.
+
+<div
+  style={{
+    textAlign: 'center',
+    backgroundColor: 'var(--color-background-alternative)',
+    padding: 32,
+  }}
+>
+  <img width="80%" src={designTokenDiagramImage} />
+</div>
+
+<br />
+<br />
+
+### **Brand colors** (tier 1)
```

Comment:
NIT:
```suggestion
### **Brand Colors** (Tier 1)
```

I noticed "Colors" in the "Theme Colors" header is also capitalized. We can also capitalize "colors" in "Component colors"

kinda like "Tiers" capitalized too, but it's not a dire desire by any means. Totally optional!

Reply:
Nice catch! I believe we should be following sentence casing so I will update to that.

Reply:
Updated in https://github.com/MetaMask/metamask-extension/pull/13657/commits/bfbb6d6cdf98c91129ae2c256ce44383365357b8